### PR TITLE
user can create a new reminder

### LINKED ISSUE #4: User can create new reminder
At the bottom of the `reminders.hbs` template, there should be a button that routes the user to `reminders/new` where users can create a new reminder.

Start looking for help [here](https://guides.emberjs.com/v2.12.0/models/defining-models/).
* The new reminder should have inputs for the `title`, `date`, and `reminders`.
* The new reminder should have a button for submitting the new reminder.
* Pressing the button should submit reminder to the database and have it display in the reminders list.

You are **not** allowed to use a component for this issue.

**Hot Tip**: There are no tests set up for this user story, you are now required to add an acceptance test and any additional tests you need before this PR will be merged in.

### PR DESCRIPTION
**Pull request by ActionJonny**
## Purpose

On click of the 'new reminder' we create two input fields - one for the title and one for the body. 
On click of the 'submit' button the users text from the two input fields will be submitted with an id and date will be applied to the new reminder.
## Approach

Users can now create their own reminders

### Learning

Learned about:
  -getProperties
  -createRecord
  -setProperties
  -transitionToRoute

### Open Questions and Pre-Merge TODOs

This pull request will close #4 

### Test coverage 

We are checking the amount of reminders before and after the user clicks the submit button.

### Follow-up tasks

- Issue #5 

### Screenshots

Screen Shot

### CODE REVIEW

**Comment by hmorri32** [comment]
@martensonbj @Tman22

File: app/routes/reminders/new.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import Ember from 'ember';
```
Comment (martensonbj) [inline app/routes/reminders/new.js]: Let's remove this file

File: app/templates/reminders/new.hbs [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<div>
+  {{input placeholder="title" value=title class="input-title"}}
+  <br>
```
Comment (martensonbj) [inline app/templates/reminders/new.hbs]: Instead of `</br>` tags lets use CSS to add a bottom margin.

File: app/templates/reminders/reminder.hbs [hunk 3]
```diff
@@ -1,8 +1,10 @@
-
-
-<div class='reminder-title'>
+<div class="reminder-title">
   {{model.title}}
 </div>
-
+<div>
+  {{model.body}}
+  <br>
```
Comment (martensonbj) [inline app/templates/reminders/reminder.hbs]: Same comment as above - use CSS here not `<br/>` tags.

**Review by martensonbj — CHANGES_REQUESTED** [review]
NIce work. Couple clean up things

**Comment by hmorri32** [comment]
closes #3

**Comment by ActionJonny** [comment]
@martensonbj @Tman22

**Comment by martensonbj** [comment]
Did you guys delete the `new` route? Is this code still working/test passing?   @jonwille @hmorri32

**Comment by ActionJonny** [comment]
We removed the unnecessary file, and everything looks good.

**Comment by martensonbj** [comment]
👍

**Pull request merged**